lib/core/include/irods_query.hpp

```diff
@@ -106,7 +106,7 @@ namespace irods {
                 return query_string_;
             }
 
-            bool page_in_flight(const int row_idx_) {
+            bool page_in_flight(const int& row_idx_) {
                 if(row_idx_ < row_cnt()) {
```

Comment:
You can replace this `if` statement with a single `return`.

Reply:
Done